entries/repo-website-overview.puml

```diff
@@ -0,0 +1,80 @@
+@startuml Repos-Sites
+
+  package .github as GIT {
+      diamond dia1
+  }
+
+  package data as DAT {
+      diamond dia2
+  }
+
+  package theme as THE {
+      diamond dia3
+  }
+
+  package community as COM {
+      object "Community.Seedcase-Project.org" as CSP {
+        Project blocks
+        Roadmap
+        Guides
+        Posts
+        Outreach
+        Code of Conduct
+      }
+  }
+
+  package decisions as DEC {
+      object "Decisions.Seedcase-Project.org" as DeCP {
+        Posts
+      }
+  }
+
+  package design as DES {
+      object "Design.Seedcase-Project.org" as DeSP {
+        Welcome
```

Comment:
```suggestion
```
Maybe not necessary to have the "Welcome" page included here? Each repo will/should have some kind of welcome/landing page and if included here, I guess we should include it for each repo in this figure?